src/NuGet.Clients/NuGet.VisualStudio.Common/PackageManagementFormat.cs

```diff
@@ -102,10 +102,17 @@ public int SelectedPackageManagementFormat
 
         public void ApplyChanges()
         {
-            _settings.AddOrUpdate(ConfigurationConstants.PackageManagementSection,
-                new AddItem(ConfigurationConstants.DefaultPackageManagementFormatKey, _selectedPackageFormat.ToString(CultureInfo.InvariantCulture)));
-            _settings.AddOrUpdate(ConfigurationConstants.PackageManagementSection,
-                new AddItem(ConfigurationConstants.DoNotShowPackageManagementSelectionKey, _showDialogValue.Value.ToString(CultureInfo.InvariantCulture)));
+            if (_selectedPackageFormat != -1)
```

Comment:
To guard against any future code changes that could use another sentinel value (like -2, etc), you could make this:
```suggestion
            if (_selectedPackageFormat > -1)
```